@planetarium/lib9c/package.json

```diff
@@ -34,5 +34,6 @@
     "buffer": "^6.0.3",
     "decimal.js": "^10.4.3",
     "uuid": "^9.0.1"
-  }
+  },
+  "packageManager": "pnpm@9.12.2+sha512.22721b3a11f81661ae1ec68ce1a7b879425a1ca5b991c975b074ac220b187ce56c708fe5db69f4c962c989452eee76c82877f4ee80f474cebd61ee13461b6228"
```

Comment:
@moreal is this OK?

Reply:
Hmm, it's not unusual to see packageManager set, but I've never seen it attached to a hash commit.